### ISSUE TITLE
Signing up and logging out as a new member is throwing  errors

### ISSUE THREAD

**Issue by BhargaviNadendla** [open-learning-exchange/planet#2804]
![image](https://user-images.githubusercontent.com/25775298/50291782-1beff800-0435-11e9-97c6-efd82d0e2673.png)

**Comment by lmmrssa** [comment]
_users/_all_docs will not be available for new user. And this is intended response.

**Comment by paulbert** [comment]
The second error is by design & our way of checking if the database is secured with an administrator password.

The first was part of the navigation layout change PR as a way to switch between new and old layouts.  This means it would not work for normal users since they cannot see the setting on `_config`.

See exact change from PR:
https://github.com/open-learning-exchange/planet/pull/2345/files#diff-e243ddc850b18eaabb654d8813bb32fbR36

**Comment by lmmrssa** [comment]
@paulbert shall we close this issue?

**Comment by paulbert** [comment]
No, I'll change the title to better reflect and note that this is very low priority
